src/components/cards/SightingMapViewCard.jsx

```diff
@@ -0,0 +1,37 @@
+import { get } from 'lodash';
+import React from 'react';
+import { useIntl, FormattedMessage } from 'react-intl';
+
+import { formatDate } from '../../utils/formatters';
+import ManyPoints from '../maps/ManyPoints';
+import Card from './Card';
+
+export default function SightingMapViewCard({
```

Comment:
since this is already rendered inside a card, we don't need a card here. i think `SightingMap` would be fine